Launcher/Classes/Constants.swift

```diff
@@ -25,6 +25,7 @@ enum Constants {
             static let flash = main.path(forResource: "flash", ofType: .bash)
             static let appDownload = main.path(forResource: "app_download", ofType: .bash)
             static let checkSimulatorType = main.path(forResource: "check_sim_type", ofType: .bash)
+            static let appInstall = main.path(forResource: "app_deploy", ofType: .bash)
```

Comment:
Why is the bash file called `app_deploy` and not `app_install`?

Reply:
Wanted to use `app_deploy` library first :) Will change